### ISSUE TITLE
🐛 BUG: 500 error page not rendering

### ISSUE THREAD

**Issue by Tc-001** [snowpackjs/astro#1420]
### What package manager are you using?

npm

### What operating system are you using?

Linux

### Describe the Bug

Error 500 nicely formatted page is not being rendered, instead returns the error message in the form of plaintext.

### Steps to Reproduce

1. `npm init astro` using any template
2. make an error
3. Bug! You will be presented with an empty page and the error printed at the top.
![image](https://user-images.githubusercontent.com/55956895/134576069-83648717-9f6b-43c9-b597-4930afc58348.png)

this can be demonstrated by using `curl` against a failing page
```
me@me ~> curl http://localhost:3000/
SyntaxError: Missing semicolon. (2:6)⏎
```
versus a working one
```
me@me ~> curl http://localhost:3000/en/introduction
<!doctype html><html dir="ltr" lang="en-us" class="initial"><head>...
```

Expected: It should render https://github.com/snowpackjs/astro/blob/main/packages/astro/src/frontend/500.astro

### Link to Minimal Reproducible Example (Optional)

https://codesandbox.io/s/brave-ptolemy-nr1f6?file=/src/pages/index.astro

**Comment by Tc-001** [comment]
From my investigation, it seems that the rendering took place in a file called "search.ts", and was added here:
https://github.com/snowpackjs/astro/pull/131/files#diff-77632f7e0b585c1410f73079ee24a75ceffea12f4deac9a82ecdcd7da2ae1469R94-R104

After that, it seems that at some point the code got refactored and `search.ts` removed. But the rendering logic was not added back in the new implementation.

**Comment by matthewp** [comment]
Thanks! As @Tc-001 pointed out the logic didn't get ported over when the routing changed. However it's changing against in the `next` branch and we do have HTML based error messages: https://github.com/snowpackjs/astro/blob/f0839c62df45c71774a6fbbe6fcbaae97f98c6bb/packages/astro/src/dev/template/error.ts#L11

Going to mark this as blocked, then we can verify it once `next` is ready.

**Comment by FredKSchott** [comment]
Thanks for the report! We believe that this has been fixed in astro v0.21, going out today. Let us know if you can still reproduce the issue after updating to v0.21 and we'll be happy to take a look.
